models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
That's a good point. I'd agree the strongest argument in favor of doing it as a JSON property is that the relationship will always be 1:1 (connection of a specific type : parent model, eg team) and that we'll only query for connections associated with a given team, like you said. 

Maybe I'll give it a shot tomorrow and see how it looks that way